### PR TITLE
prow: descheduler: deploy 1.18.2 cluster and run make test-e2e

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

File: config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-master.yaml [hunk 1]
```diff
@@ -52,3 +52,32 @@ presubmits:
...
+    - master
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200710-6b3b6fe-master
+        command:
+        # generic runner script, handles DIND, bazelrc for caching, etc.
+        - runner.sh
+        env:
+        - name: KUBERNETES_VERSION
+          value: "v1.18.2"
```
Comment (damemi) [inline config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-master.yaml:77]: we'll want to update this to 1.19 after the rebase right?

Comment (ingvagabund) [reply config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-master.yaml:77]: yeah, this is basically for testing to see how it's wired.

**Comment by ingvagabund** [comment]
> Is there any way for us to test it besides merging and seeing what happens?

I wish I knew a better (and practical) way.

**Comment by damemi** [comment]
@ingvagabund looks like it failed to get the kubeconfig from the expected location: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_descheduler/338/pull-descheduler-test-e2e-k8s-master/1282722202692620289#1:build-log.txt%3A11

The test expects it in /tmp/admin.conf: https://github.com/kubernetes-sigs/descheduler/blob/c5c8e76a870318d074ff2c4652b48e334efa5620/test/e2e/e2e_test.go#L152

and the current travis script sets it up: https://github.com/kubernetes-sigs/descheduler/blob/master/.travis.yml#L24

Maybe we could just change the e2e to use `os.Getenv("KUBECONFIG")`?

**Pull request merged**